SCloudRandomizer/MainViewController.m

```diff
@@ -7,24 +7,34 @@
 //
 
 #import "MainViewController.h"
+#import "Utility.h"
+#import "Track.h"
 
+const double LoggedOutBackgroundImage_Opacity = 0.7;
```

Comment:
Its good objective-C practice to add the 'static' keyword in front of the constants.